phpcs.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="alleyinteractive/wp-type-extensions" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+    <description>PHP_CodeSniffer standard for alleyinteractive/wp-type-extensions.</description>
+
+    <file>src/</file>
+    <file>tests/</file>
+    <exclude-pattern>vendor/</exclude-pattern>
+
+    <!--
+    Pass some flags to PHPCS:
+      p flag: Show progress of the run.
+      s flag: Show sniff codes in all reports.
+    -->
+    <arg value="ps"/>
+
+    <!-- Use the Alley standard. -->
+    <rule ref="Alley-Interactive"/>
+
+    <!-- Check up to N files simultaneously. -->
+    <arg name="parallel" value="75"/>
+
+    <!-- Set severity to 1 to see everything that isn't effectively turned off. -->
+    <arg name="severity" value="1"/>
+
+    <!-- Set the allowed text domains for i18n. -->
+    <rule ref="WordPress.WP.I18n">
+        <properties>
+            <property name="text_domain" type="array" value="alley"/>
```

Comment:
```suggestion
            <property name="text_domain" type="array" value="wp-type-extensions"/>
```

Reply:
maybe not even applicable...

Reply:
Hmm. I've been using `alley` in public packages (e.g. [here](https://github.com/alleyinteractive/wp-match-blocks/blob/367e1376ff7d0dd436e2ab4b1011e0c3da48f4f9/phpcs.xml) and [here](https://github.com/alleyinteractive/wp-find-one/blob/ea525cc7c14c4e247aef4e69433d5e18eab24d2a/phpcs.xml#L28)). Is there a documented standard for this?